tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -302,3 +331,54 @@ def test_plot_pareto_front_invalid_axis_order(
             include_dominated_trials=include_dominated_trials,
             axis_order=invalid_axis_order,
         )
+
+
+def test_plot_pareto_front_targets_without_target_names() -> None:
+    study = optuna.create_study(directions=["minimize", "minimize", "minimize"])
+    with pytest.raises(ValueError):
+        plot_pareto_front(
+            study=study,
+            target_names=None,
+            targets=lambda t: (t.values[0], t.values[1], t.values[2]),
+        )
+
+
+@pytest.mark.parametrize(
+    "targets",
+    [
+        lambda t: (t.values[0]),
+        lambda t: (t.values[0],),
+        lambda t: (t.values[0], t.values[1], t.values[2], t.values[3]),
+    ],
+)
+def test_plot_pareto_front_invalid_targets(
+    targets: Optional[Callable[[FrozenTrial], Sequence[float]]]
+) -> None:
+    study = optuna.create_study(directions=["minimize", "minimize", "minimize", "minimize"])
+    study.enqueue_trial({"w": 1, "x": 1, "y": 1, "z": 1})
+    study.enqueue_trial({"w": 0, "x": 1, "y": 0, "z": 1})
+    study.enqueue_trial({"w": 1, "x": 1, "y": 1, "z": 0})
+    study.optimize(
+        lambda t: [
+            t.suggest_int("w", 0, 1),
+            t.suggest_int("x", 0, 1),
+            t.suggest_int("y", 0, 1),
+            t.suggest_int("z", 0, 1),
+        ],
+        n_trials=3,
+    )
+    with pytest.raises(ValueError):
```

Comment:
ditto: specifying `match` argument of `pytest.raises`.
```suggestion
    with pytest.raises(ValueError):
```